claimManagement/src/main/java/org/openimis/imisclaims/MainActivity.java

```diff
@@ -66,7 +77,9 @@ public class MainActivity extends ImisActivity {
 
     final String VersionField = "AppVersionEnquire";
     NotificationManager mNotificationManager;
+    public File f;
```

Comment:
Please move this field variable (public File f) to local (onActivityResult). Also, please rename it to something more descriptive.

Reply:
Changed, will mark as solved.